### ISSUE TITLE
Outlook to Outlook sync

### ISSUE THREAD

**Issue by cavemandaveman** [phw198/OutlookGoogleCalendarSync#1410]
**Are you already using OGCS?**
2.9.3

**Describe the question you have**
Scenario: 2 Outlook accounts, 1 Google account
Desire: All events shown on all calendars

Is it possible to sync Outlook A to Google AND Outlook B, as well as sync Outlook B to Google AND Outlook A?

I tried to do this with multiple profiles and two-way sync, but ended up with duplicates all over the place.

So, this partly seeing if anyone has been successful with this kind of setup, partly a feature request to enable an Outlook to Outlook sync which I think would solve this problem.

**Comment by phw198** [comment]
Yes, "Outlook A <-> Google AB Calendar <-> Outlook B" should be a possible configuration with 2x bi-directional sync profiles.

I suggest you start with empty calendars and build the configuration up. It's possible if you are trying with existing calendars that were already synced things have got pretty messy!

Note, you would need to have the "Merge" option turned on.
